svencoop.asl

```diff
@@ -1,38 +1,52 @@
-//Sven Co-op LiveSplit Auto-Splitter - Edit Layout - Scriptable Auto Splitter
 //Thanks for supporting the project with code: BenInSweden and Chillu
-//Works only with Sven Co-op 15 April 2017 version

```

Comment:
Why was this removed? The program still only works with 15 april version, right?

Reply:
I think since the version is written in the LiveSplit and in README.md, then i don’t think it is worth mentioning in the description of the .asl script too.